boards.txt

```diff
@@ -15852,6 +15852,473 @@ deneyapkart.menu.EraseFlash.all.upload.erase_cmd=-e
+deneyapmini.menu.PartitionScheme.huge_app=Huge APP (3MB No OTA/1MB SPIFFS)
+deneyapmini.menu.PartitionScheme.huge_app.build.partitions=huge_app
+deneyapmini.menu.PartitionScheme.huge_app.upload.maximum_size=3145728
+deneyapmini.menu.PartitionScheme.min_spiffs=Minimal SPIFFS (1.9MB APP with OTA/190KB SPIFFS)
+deneyapmini.menu.PartitionScheme.min_spiffs.build.partitions=min_spiffs
+deneyapmini.menu.PartitionScheme.min_spiffs.upload.maximum_size=1966080
+deneyapmini.menu.PartitionScheme.fatflash=16M Flash (2MB APP/12.5MB FATFS)
+deneyapmini.menu.PartitionScheme.fatflash.build.partitions=ffat
+deneyapmini.menu.PartitionScheme.fatflash.upload.maximum_size=2097152
+deneyapmini.menu.PartitionScheme.app3M_fat9M_16MB=16M Flash (3MB APP/9.9MB FATFS)
+deneyapmini.menu.PartitionScheme.app3M_fat9M_16MB.build.partitions=app3M_fat9M_16MB
+deneyapmini.menu.PartitionScheme.app3M_fat9M_16MB.upload.maximum_size=3145728
+
+deneyapmini.menu.CPUFreq.240=240MHz (WiFi)
+deneyapmini.menu.CPUFreq.240.build.f_cpu=240000000L
+deneyapmini.menu.CPUFreq.160=160MHz (WiFi)
+deneyapmini.menu.CPUFreq.160.build.f_cpu=160000000L
+deneyapmini.menu.CPUFreq.80=80MHz (WiFi)
+deneyapmini.menu.CPUFreq.80.build.f_cpu=80000000L
+deneyapmini.menu.CPUFreq.40=40MHz
+deneyapmini.menu.CPUFreq.40.build.f_cpu=40000000L
+deneyapmini.menu.CPUFreq.20=20MHz
+deneyapmini.menu.CPUFreq.20.build.f_cpu=20000000L
+deneyapmini.menu.CPUFreq.10=10MHz
+deneyapmini.menu.CPUFreq.10.build.f_cpu=10000000L
+
+deneyapmini.menu.FlashMode.qio=QIO
+deneyapmini.menu.FlashMode.qio.build.flash_mode=dio
+deneyapmini.menu.FlashMode.qio.build.boot=qio
+deneyapmini.menu.FlashMode.dio=DIO
+deneyapmini.menu.FlashMode.dio.build.flash_mode=dio
+deneyapmini.menu.FlashMode.dio.build.boot=dio
+deneyapmini.menu.FlashMode.qout=QOUT
+deneyapmini.menu.FlashMode.qout.build.flash_mode=dout
+deneyapmini.menu.FlashMode.qout.build.boot=qout
+deneyapmini.menu.FlashMode.dout=DOUT
+deneyapmini.menu.FlashMode.dout.build.flash_mode=dout
+deneyapmini.menu.FlashMode.dout.build.boot=dout
+
+deneyapmini.menu.FlashFreq.80=80MHz
+deneyapmini.menu.FlashFreq.80.build.flash_freq=80m
+deneyapmini.menu.FlashFreq.40=40MHz
+deneyapmini.menu.FlashFreq.40.build.flash_freq=40m
+
+deneyapmini.menu.FlashSize.4M=4MB (32Mb)
+deneyapmini.menu.FlashSize.4M.build.flash_size=4MB
+deneyapmini.menu.FlashSize.8M=8MB (64Mb)
+deneyapmini.menu.FlashSize.8M.build.flash_size=8MB
+deneyapmini.menu.FlashSize.8M.build.partitions=default_8MB
+deneyapmini.menu.FlashSize.2M=2MB (16Mb)
+deneyapmini.menu.FlashSize.2M.build.flash_size=2MB
+deneyapmini.menu.FlashSize.2M.build.partitions=minimal
+deneyapmini.menu.FlashSize.16M=16MB (128Mb)
+deneyapmini.menu.FlashSize.16M.build.flash_size=16MB
+
+deneyapmini.menu.UploadMode.default=Internal USB
+deneyapmini.menu.UploadMode.default.upload.use_1200bps_touch=true
+deneyapmini.menu.UploadMode.default.upload.wait_for_upload_port=true
+deneyapmini.menu.UploadMode.default.upload.mode=default_reset
+deneyapmini.menu.UploadMode.cdc=UART0
```

Comment:
CDC is USB, not UART0. You can just switch the order of items to have USB as default